### ISSUE TITLE
关于lightStereo里面采样mobilenetv2的训练过程

### ISSUE THREAD

**Issue by Hlll-huang** [XiandaGuo/OpenStereo#134]
您好，我看立体匹配算法lightstereo使用了mobilenetv2网络作为backbone的特征提取，想进一步问问在训练的时候，这部分的权重参数是冻结的，还是只是训练初始化的时候使用mobilenet的权重。整个训练过程都是在优化参数的呢

**Comment by zhcm** [comment]
初始化时加载mobilenet预训练权重,训练过程使用相同的学习率优化所有参数 @Hlll-huang

**Comment by zhcm** [comment]
Initialize with MobileNet pre-trained weights, and use the same learning rate to optimize all parameters during the training process. @Hlll-huang

**Comment by Hlll-huang** [comment]
thanks



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年9月23日(周一) 下午4:16
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [XiandaGuo/OpenStereo] 关于lightStereo里面采样mobilenetv2的训练过程 (Issue #134)




 
Initialize with MobileNet pre-trained weights, and use the same learning rate to optimize all parameters during the training process. @Hlll-huang

**Comment by luoxieyang** [comment]
楼主您好，我关于lightstereo在kitti测试集上测试的一些小问题可以方便回答吗
